code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -269,13 +281,28 @@
 	recently_dead_ckeys -= ckey
 
 /obj/machinery/capture_the_flag/proc/spawn_team_member(client/new_team_member)
+	var/datum/outfit/chosen_class
+	if(ctf_gear.len == 1) //no choices to make
+		chosen_class = ctf_gear[1]
+	if(ctf_gear.len > 3) //a lot of choices, so much that we can't use a basic alert
```

Comment:
would be cool if this were a radial instead 😏

Reply:
hhgaeafe aep hehu rfae[hget